src/pages/Login/LoginForm.tsx

```diff
@@ -77,9 +89,30 @@ export const LoginForm = () => {
             {...register('password')}
           />
         </Box>
-        <Button type="submit" disabled={!isValid}>
+        <Button
+          type="submit"
+          disabled={!isValid}
+          loading={mutationResult.loading}
+        >
           {messages.login.loginFrom.submitButton.text}
         </Button>
+        {errMsg ? (
+          <Alert
+            icon={<ErrorAlertIcon width={18} height={18} />}
+            title={errMsg}
+            color="red"
+            sx={theme => ({
+              'color': theme.colors.errorDefault[6],
+              'padding': '12px 14px',
+              '.mantine-rtl-Alert-title': {
+                fontSize: 12,
+                fontWeight: 400,
+              },
+            })}
+          >
+            {''}
```

Comment:
This one too

Reply:
if u mean this `{''}` The children of  `Alert` component is required. but based on design we have to use title for our error messages.